README.md

```diff
@@ -1,5 +1,5 @@
 # React add TODO form
-- Replace `<your_account>` with your Github username in the [DEMO LINK](https://<your_account>.github.io/react_add-todo-form/)
+- Replace `<your_account>` with your Github username in the [DEMO LINK](https://naziamoff.github.io/react_add-todo-form/)
```

Comment:
<img width="1476" alt="image" src="https://user-images.githubusercontent.com/38065883/175115785-ec0e5f4e-e5c8-460d-bbdf-3f97d067932c.png">
Check todo.trim().length before adding it